www/routes/_middleware.ts

```diff
@@ -61,6 +61,7 @@ function ga4(
 
     // Create basic report.
     const measurementId = GA4_MEASUREMENT_ID;
+    // @ts-ignore GA4Report doesn't even use the localAddress parameter
```

Comment:
Pretty unfortunate but the typing of this module is just wrong.

Reply:
All good, happens from time to time with third party libraries.